opensearch/src/main/java/org/opensearch/sql/opensearch/planner/logical/OpenSearchLogicalIndexScan.java

```diff
@@ -67,14 +71,15 @@ public OpenSearchLogicalIndexScan(
       Expression filter,
       Set<ReferenceExpression> projectList,
       List<Pair<Sort.SortOption, Expression>> sortList,
-      Integer limit, Integer offset) {
+      Integer limit, Integer offset, String highlightField) {
```

Comment:
How the OpenSearchLogicalIndexScan is initialized with highlightFields?
HighlightOperator is no-op operator, which seems useless.
Have u consider just use logical expression to represent highlight fields?